lib/MusicBrainz/Server/Controller/Edit.pm

```diff
@@ -278,6 +278,31 @@ sub subscribed_editors : Local RequireAuth {
     load_everything_for_edits($c, $edits);
 }
 
+sub notes_received : Path('/edit/notes-received') RequireAuth {
+    my ($self, $c) = @_;
+
+    # Log when the editor loaded the page, so that we know when to notify them
+    # again about new edits (see Data::EditNote::add_note).
+    my $redis = $c->model('MB')->context->redis;
+    my $notes_viewed_key = $c->user->name . '|edit_notes_received_last_viewed';
```

Comment:
I think we should name the keys a bit differently. Catalyst uses `session:$sid`, `expires:$sid`, `flash:$sid` (where `$sid` stands for the session ID), so perhaps `edit_notes_received_last_viewed:$user` and `edit_notes_received_last_updated:$user`? I know the remember-me tokens use `$user|$token` without indication what they are about, but I’ve thought before that `remember-me:$user:$token` (or something) might be better; it’s just complicated to change without breaking existing sessions, so I haven’t done it yet.